packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -1903,6 +1903,29 @@ describe("normalizeAndValidateConfig()", () => {
 					  - Expected \\"experimental_assets.directory\\" to be a non-empty string."
 				`);
 			});
+
+			it("should error on invalid additional fields", () => {
+				const expectedConfig = {
+					experimental_assets: {
+						directory: "./public",
+						invalid_field_1: "this is invalid",
+						invalid_field_2: "this is invalid too",
+					},
```

Comment:
Will it fail with a different error once `experimental_assets` is renamed on #6802?

Reply:
that PR needs to refactor these changes